datakyt/db/dump_db.py

```diff
@@ -0,0 +1,47 @@
+import subprocess
+import logging
+from datetime import datetime
+
+logger = logging.getLogger(__name__)
+logging.basicConfig(level=logging.DEBUG,
+                    format='%(asctime)s.%(msecs)03d[%(process)d:%(thread)d:%(name)s:%(lineno)d] %(levelname)s %('
+                           'message)s',
+                    datefmt='%Y-%m-%d %I:%M:%S')
+
+
+def backup_postgres_db(host, database_name, port, user, password, backup_file_path):
```

Comment:
Please, change the order of  parameters so one-group parameters will be passed one by one: database_name, host, port,  user, password, backup_file_path

Reply:
Okey